.rhcicd/pr_check_fp.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -exv
+
+# Clowder config
+export APP_NAME="policies"
+export COMPONENT_NAME="policies-ui-backend"
+export IMAGE="quay.io/cloudservices/iqe-tests:latest"
```

Comment:
If floorist is getting installed in `iqe-tests:policies` you can still use it here.

Reply:
The iqe-floorist-plugin does not seem to be part of the :policices tag. I could add the floorist plugin as a dependency in the policices plugin and then use the :policices tag.

Reply:
I updated the MR to use iqe-tests:policies after adding it to the image.